parsec/core/fs/storage/manifest_storage.py

```diff
@@ -11,6 +13,8 @@
 
 logger = get_logger()
 
+EMPTY_PATTERN = r"^/b$"  # Do not match anything
```

Comment:
PR author hates him: a young programmer found a pattern that match the "do not match anything" regex !
```python
>>> re.match(r'^/b$', '/b')
<_sre.SRE_Match object; span=(0, 2), match='/b'>
```

Reply:
Good catch! It's a typo, it should be [`r'^\b$'`](https://stackoverflow.com/a/2302992/2846140)